packages/code-snippet/src/CodeSnippetWidget.tsx

```diff
@@ -72,31 +74,66 @@ class CodeSnippetDisplay extends React.Component<ICodeSnippetDisplayProps> {
       const documentWidget = widget as DocumentWidget;
       const fileEditor = (documentWidget.content as FileEditor).editor;
 
-      // Wrap in code block if inserting snippet into a markdown file
       if (PathExt.extname(widget.context.path) === '.md') {
+        // Wrap snippet into a code block when inserting it into a markdown file
         fileEditor.replaceSelection(
           '```' + snippet.language + '\n' + snippetStr + '\n```'
         );
+
+        // TODO: Change how python file editor is checked below to be more generic
```

Comment:
should probably move this into the if statement (indentation just looks off having it here)